resources/cdk/aurora_serverless_app/README.md

```diff
@@ -0,0 +1,137 @@
+# Aurora serverless sample application resources
+
+## Overview
+
+Creates AWS resources for Amazon Aurora sample applications. The scripts in this
+example create the following resources:
+ 
+* An AWS Secrets Manager secret that contains administrator credentials in a format 
+that can be used by an Aurora MySQL database. 
+* An Aurora MySQL database configured to use the credentials from the secret.
+
+## ⚠️ Important
+
+* Running this code might result in charges to your AWS account. 
+* This code is not tested in every AWS Region. For more information, see [AWS Regional Services](https://aws.amazon.com/about-aws/global-infrastructure/regional-product-services).
+
+## Deploying resources
+
+You can use the AWS Cloud Development Kit (AWS CDK) or the AWS Command Line Interface
+(AWS CLI) to deploy and destroy the resources for this example.
+
+### Deploying with the AWS CDK
+
+To deploy with the AWS CDK, you must install [Node.js](https://nodejs.org) and the 
+[AWS CDK](https://docs.aws.amazon.com/cdk/v2/guide/getting_started.html).
+
+This example was built and tested with AWS CDK 2.25.0.
+
+Deploy AWS resources by running the following at a command prompt in the
+`resources/cdk/aurora_serverless_app` folder:
+
+```
+npm install
+cdk deploy
+```
+
+The stack takes a few minutes to deploy. When it completes, it prints output like 
+the following:
+
+```
+Outputs:
+doc-example-aurora-app.ClusterArn = arn:aws:rds:us-west-2:0123456789012:cluster:doc-example-aurora-app-docexampleauroraappcluster-1bqmf5EXAMPLE
+doc-example-aurora-app.DbName = auroraappdb
+doc-example-aurora-app.SecretArn = arn:aws:secretsmanager:us-west-2:0123456789012:secret:docexampleauroraappsecret8B-rEHdtEXAMPLE-111222
+```
+
+You can destroy the resources by running the following:
+
+```
+cdk destroy
+```
+
+### Deploying with the AWS CLI 
+
+To deploy with the AWS CLI you must first install the 
+[AWS CLI](https://docs.aws.amazon.com/cli/latest/userguide/getting-started-install.html).
+
+Deploy AWS resources by running the following at a command prompt in the 
+`resources/cdk/aurora_serverless_app` folder:
+
+```
+aws cloudformation create-stack --template-body file://setup.yaml --stack-name YOUR_STACK_NAME
+```
+
+The stack takes a few minutes to deploy. You can check status by running the following:
+
+```
+aws cloudformation describe-stacks --stack-name YOUR_STACK_NAME
+```
+
+When the stack is ready, it shows `StackStatus` of `CREATE_COMPLETE`.
+
+You can get the outputs from the stack by running the following:
+
+```
+aws cloudformation describe-stacks --stack-name STACK_NAME --query Stacks[0].Outputs --output text
+```
+
+This results in output like the following: 
+
+```
+SecretArn  arn:aws:secretsmanager:us-west-2:0123456789012:secret:docexampleauroraappsecret8B-6N2njEXAMPLE-111222
+ClusterArn arn:aws:rds:us-west-2:0123456789012:cluster:aurora-test-stack-docexampleauroraappcluster12345-kh39pEXAMPLE
+DbName     auroraappdb
+```
+
```

Comment:
Suggest adding a title here, so that we can link to it from the READMEs for the cross-service examples that uses it. Maybe 'Destroying resources'. 
Also suggest adding note:
Note: If any of the resources have been altered, you must manually delete them via the AWS Console.

Reply:
I rearranged and added the title for linking, but I omitted the note because it doesn't seem to apply in this case (i.e., if you create a table in the DB you can still destroy with CDK/CLI).